plugins/talk-plugin-moderation-actions/client/actions.js

```diff
@@ -0,0 +1,10 @@
+import {OPEN_TOOLTIP, CLOSE_TOOLTIP} from './constants';
```

Comment:
According to wikipedia, a `tooltip `is sth that shows when hovering over it, so I don't think this is the correct term here. I'd just call it _menu_.